src/app/modules/sgid/__tests__/sgid.controller.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { err, errAsync, ok, okAsync } from 'neverthrow'
+import { mocked } from 'ts-jest/utils'
+
+import config from 'src/app/config/config'
+import * as RealFormService from 'src/app/modules/form/form.service'
+import { MOCK_COOKIE_AGE } from 'src/app/modules/myinfo/__tests__/myinfo.test.constants'
+
+import expressHandler from 'tests/unit/backend/helpers/jest-express'
+
+import { ApplicationError } from '../../core/core.errors'
+import { FormNotFoundError } from '../../form/form.errors'
+import * as SgidController from '../sgid.controller'
+import {
+  SgidFetchAccessTokenError,
+  SgidFetchUserInfoError,
+  SgidInvalidStateError,
+} from '../sgid.errors'
+import { sgidService as realSgidService } from '../sgid.service'
+
+import {
+  MOCK_AUTH_CODE,
+  MOCK_COOKIE_SETTINGS,
+  MOCK_DESTINATION,
+  MOCK_JWT,
+  MOCK_REMEMBER_ME,
+  MOCK_SGID_FORM,
+  MOCK_SP_FORM,
+  MOCK_STATE,
+  MOCK_TARGET,
+  MOCK_TOKEN_RESULT,
+  MOCK_USER_INFO,
+} from './sgid.test.constants'
+
+jest.mock('../sgid.service')
+const sgidService = mocked(realSgidService, true)
```

Comment:
nit: PascalCase the service name (`SgidService`)?

Reply:
It isn't a class definition though, but okay.